packages/steps/src/step.vue

```diff
@@ -137,24 +137,25 @@ export default {
   methods: {
     updateStatus(val) {
       const prevChild = this.$parent.$children[this.index - 1];
+      let currentProcessStatus = false;
```

Comment:
Your pr has fixed issue #13068. But I find codes of Step component is hard to understand.